packages/access-control/lib/access-control.js

```diff
@@ -64,11 +64,64 @@ const parseAccess = ({
   }
 };
 
+const parseAccess = ({
+  schemaNames,
+  accessTypes,
+  access,
+  defaultAccess,
+  onGranularParseError,
+  validateGranularType,
+}) => {
+  // Check that none of the schemaNames match the accessTypes
+  if (intersection(schemaNames, accessTypes).length > 0) {
+    throw new Error(
+      `Invalid schemaNames: ${JSON.stringify(intersection(schemaNames, accessTypes))}`
+    );
+  }
+
+  const N = intersection(Object.keys(access), schemaNames).length;
```

Comment:
What is `N`? Could we make this at least 3 characters long so searching for it is easier?
```suggestion
  const totalSchemasWithAccess = intersection(Object.keys(access), schemaNames).length;
```